internal/devbox/generate/devcontainer_util.go

```diff
@@ -140,32 +140,33 @@ func (g *Options) CreateDevcontainer(ctx context.Context) error {
 	return err
 }
 
-func CreateEnvrc(ctx context.Context, path string, envFlags devopt.EnvFlags) error {
+func CreateEnvrc(ctx context.Context, opts devopt.EnvrcOpts) error {
 	defer trace.StartRegion(ctx, "createEnvrc").End()
 
 	// create .envrc file
-	file, err := os.Create(filepath.Join(path, ".envrc"))
+	file, err := os.Create(filepath.Join(opts.EnvrcDir, ".envrc"))
 	if err != nil {
 		return err
 	}
 	defer file.Close()
 
 	flags := []string{}
 
-	if len(envFlags.EnvMap) > 0 {
-		for k, v := range envFlags.EnvMap {
+	if len(opts.EnvMap) > 0 {
+		for k, v := range opts.EnvMap {
 			flags = append(flags, fmt.Sprintf("--env %s=%s", k, v))
 		}
 	}
-	if envFlags.EnvFile != "" {
-		flags = append(flags, fmt.Sprintf("--env-file %s", envFlags.EnvFile))
+	if opts.EnvFile != "" {
+		flags = append(flags, fmt.Sprintf("--env-file %s", opts.EnvFile))
 	}
 
 	t := template.Must(template.ParseFS(tmplFS, "tmpl/envrc.tmpl"))
 
 	// write content into file
 	return t.Execute(file, map[string]string{
 		"Flags": strings.Join(flags, " "),
+		"Dir":   opts.ConfigDir,
```

Comment:
Name this `ConfigDir` to reduce ambiguity. 

Compute relative path here instead of `generate.go`

Reply:
Make empty if `envrcDir` is not specified.